.github/workflows/ci.yml

```diff
@@ -170,27 +170,18 @@ jobs:
           key: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-${{ hashFiles('Cargo.lock', 'build.rs', 'git_submodule_status.txt') }}
           restore-keys: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-
 
-      - name: Install and start sccache
-        shell: pwsh
+      - name: Install sccache
+        uses: mozilla-actions/sccache-action@v0.0.8
+
+      - name: Start sccache
         env:
+          # Start sccache server instead of using `RUSTC_WRAPPER` environment variable (recommended by `sccache-action`).
+          # Because it cannot compile `syn` lib correctly on windows.
           SCCACHE_DIR: ${{ github.workspace }}/target/sccache
-          SCCACHE_CACHE_SIZE: 128M
+          SCCACHE_CACHE_SIZE: 1G
           SCCACHE_IDLE_TIMEOUT: 0
         run: |
-          $version = "0.2.12"
-          $platform =
-            @{ "macOS"   = "x86_64-apple-darwin"
-               "Linux"   = "x86_64-unknown-linux-musl"
-               "Windows" = "x86_64-pc-windows-msvc"
-             }.${{ runner.os }}
-          $basename = "sccache-$version-$platform"
-          $url = "https://github.com/mozilla/sccache/releases/download/" +
-                 "$version/$basename.tar.gz"
-          cd ~
-          curl -LO $url
-          tar -xzvf "$basename.tar.gz"
-          . $basename/sccache --start-server
-          echo "$(pwd)/$basename" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          sccache --start-server
```

Comment:
Based on my knowledge, this is simply starting the sccache local service. But it doesn't add the `RUSTC_WRAPPER` environment variable when running `cargo build` command, I don't know why the sccache still works (because the summary shows the cache hit rate is `8%`)

Reply:
In this commit: https://github.com/denoland/rusty_v8/pull/1748/commits/b21c50b8c07d65d942a3fae2f087d554b99f01e4, I explicitly set `RUSTC_WRAPPER` for linux/macos (since in my previous testing, it is not working for windows, so I disable it on windows).

I want to verify, if `sccache --start-server` works just correctly. Or using the `RUSTC_WRAPPER` is the correct way to involve sccache when building it.

The runs is: https://github.com/linrongbin16/rusty_v8/actions/runs/14054651247